cmd/tnf/generate/catalog/catalog.go

```diff
@@ -291,8 +299,54 @@ func runGenerateMarkdownCmd(_ *cobra.Command, _ []string) error {
 
 // Execute executes the "catalog" CLI.
 func NewCommand() *cobra.Command {
+	generatefeedbackjs.Flags().StringVarP(
+		&feedbackjson, "feedback", "f", "",
+		"feedback.json file to convert it to js file, output is js ",
+	)
+
+	err := generatefeedbackjs.MarkFlagRequired("feedback")
+	if err != nil {
+		return nil
+	}
+	generateCmd.AddCommand(generatefeedbackjs)
+
 	generateCmd.AddCommand(markdownGenerateCmd)
 
 	generateCmd.AddCommand(markdownGenerateClassification)
 	return generateCmd
 }
+
+func runGeneratefeedbackjs(_ *cobra.Command, _ []string) error {
+	feedbackjsonFileTextPtr := &feedbackjson
+	dat, err := os.ReadFile(*feedbackjsonFileTextPtr)
+	if err != nil {
+		log.Fatalf("Error reading claim file :%v", err)
+	}
+	var obj map[string]interface{}
+	err = json.Unmarshal(dat, &obj)
+	if err != nil {
+		fmt.Println("Error:", err)
```

Comment:
There's no need to use these `fmt.Println()` before returning the error, as the error is always printed. Just create a helpful fmt.Errorf("...") message.